lib/grape/dsl/parameters.rb

```diff
@@ -165,9 +165,11 @@ def all_or_none_of(*attrs)
       # @raise Grape::Exceptions::UnknownParameter if `attr` has not been
       #   defined in this scope yet
       # @yield a parameter definition DSL
-      def given(attr, &block)
-        fail Grape::Exceptions::UnknownParameter.new(attr) unless declared_param?(attr)
-        new_lateral_scope(dependent_on: attr, &block)
+      def given(*attrs, &block)
+        attrs.each do |attr|
+          fail Grape::Exceptions::UnknownParameter.new(attr) unless declared_param?(attr)
+          new_lateral_scope(dependent_on: attr, &block)
```

Comment:
@rnubel 
I made a change to fix this. Now the behavior is this:
`given :a, :b` will now only do the validation if both `:a` and `:b` are both given, which would mimic the nested structure as well